server/controllers/profile.js

```diff
@@ -38,4 +36,23 @@ exports.loadProfile = asyncHandler(async (req, res, next) => {
       profile: profile,
     },
   });
+});
+
+// @route GET /profile/sitter
+// @desc Get all profile data
+// @access Private
+exports.getAllSitters = asyncHandler(async (req, res) => {
+  const result = [];
+  const profiles = await Profile.find({ isSitter: true, price: { $ne: null } })
+  profiles.forEach((profile) => {
```

Comment:
Alternatively, we can use a mongoose query to filter through the db based on location